x/gov/tally_test.go

```diff
@@ -19,9 +19,15 @@ var (
 
 func createValidators(t *testing.T, stakeHandler sdk.Handler, ctx sdk.Context, addrs []sdk.ValAddress, coinAmt []int64) {
 	require.True(t, len(addrs) <= len(pubkeys), "Not enough pubkeys specified at top of file.")
+
 	dummyDescription := stake.NewDescription("T", "E", "S", "T")
+	dummyCommission := stake.NewCommission(sdk.ZeroDec(), sdk.ZeroDec(), sdk.ZeroDec())
```

Comment:
Can we make dummyDesc and dummyComm var's at the top of the file? (non-blocking, just a suggestion)

Reply:
👍